### DISCUSSION TITLE
Re-verify github education academic state issue

### DISCUSSION THREAD

**Discussion by abdulrahman-hegazy** [community/community#28390] [GitHub Education]
Hi everyone,

I subscribed to GitHub student pack one month before now.

and in the last week they asked me for re-verify my academic state

But i faced issue i will explain it in the these few lines

Only take photo from camera for your academic state option appears

My academic state is written in Arabic

This re-verify use ai to verifying my academic state and he is rejected it becase my college name is not appears but it is appears, and that make sense becase he searching for english name but there is nothing in english

And i expect if this issue solved he will ask for my name after it because it in arabic too

How can i solve this ?

![1659641633678](https://user-images.githubusercontent.com/105675263/182937694-9231d02a-89a6-4634-a865-bbb72cd2c3ba.jpg)

**Comment by Zoltus** [comment]
I think you should contact support for manual approval. Or if you can get pdf file or something from ur school site and translate it to english possibly. Good luck!

**Comment by manasesjesus** [comment]
have you added your university email to your GitHub settings? 
you may also ask your university to issue you a certificate in english

**Reply by abdulrahman-hegazy** [reply]
Yeah I added my university email,

It's impossible to issue a certificate in English in my university,

This problem will not be solved

Their image recognition code sucks. Pretty obvious that they don't want to pay anyone to actually verify students. Bad form Microsoft

**Comment by NightMachinery** [comment]
I added the needed details in English to a white section concatenated to the scanned image. This could pass the AI check.

**Comment by moncefdhk999** [comment]
[Uploading PPT_AlphaCopper_V8_11July22.pdf…]()

**Reply by ankitkat042** [reply]
> Uploading PPT_AlphaCopper_V8_11July22.pdf…

did it work?
